.github/workflows/build-and-push.yml

```diff
@@ -1,5 +1,6 @@
 name: Rebuild & push images
 on:
+  workflow_dispatch:
```

Comment:
nice, can you add it also to [the template](https://github.com/packit/.github/blob/main/workflow-templates/build-and-push.yml)?

Reply:
:+1:  packit/.github#4